example/MainWindow.cpp

```diff
@@ -61,6 +61,14 @@ MainWindow::MainWindow(QWidget *parent) :
       ToolWindowManager::AreaReference(ToolWindowManager::TopOf,
                                        ui->toolWindowManager->areaOf(toolWindows[4])));
 
+#if 0
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[0], ToolWindowManager::AlwaysDisplayFullTabs);
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[1], ToolWindowManager::AlwaysDisplayFullTabs);
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[2], ToolWindowManager::AlwaysDisplayFullTabs);
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[3], ToolWindowManager::AlwaysDisplayFullTabs);
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[4], ToolWindowManager::AlwaysDisplayFullTabs);
+  ui->toolWindowManager->setToolWindowProperties(toolWindows[5], ToolWindowManager::AlwaysDisplayFullTabs);
+#endif
```

Comment:
Can you remove this (I guess you were using it for testing?). I think the example shouldn't have any stale/outdated code in it.

Reply:
Removed it in next commit.